web/pageheader.jspf

```diff
@@ -16,14 +16,13 @@ information: Portions Copyright [yyyy] [name of copyright owner]
 
 CDDL HEADER END
 
-Copyright (c) 2007, 2010, Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2007, 2017, Oracle and/or its affiliates. All rights reserved.
 
 --%><% 
 /* ---------------------- pageheader.jspf start --------------------- */
 {
     cfg = PageConfig.get(request);
 %>
-<a href="<%= request.getContextPath() %>/"><span id="MastheadLogo"></span></a>
```

Comment:
The alternative would be to replace the 'Home' link with a small logo, however the concept of clicking on OpenGrok logo leading to the home of the webapp is a bit strange as one could expect to lead to the project page on Github. So leaving it for next time.